src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,14 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null;
```

Comment:
```suggestion
  user: User;
```
You already check it via conditional rendering